src/core/videosource.h

```diff
@@ -29,6 +29,11 @@ extern "C" {
 #include <libavutil/stereo3d.h>
 #include <libavutil/display.h>
 #include <libavutil/mastering_display_metadata.h>
+
+#include <ffmscompat.h>
```

Comment:
Should be `#include "ffmscompat.h"` since it's in-tree.

Reply:
Right, my bad.